mars/lib/filesystem/_oss_lib/import_err_handler.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2021 Alibaba Group Holding Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class ImportErrorHandler:
+	def __init__(self, name: str):
+		self._name = name
+	
+	def __getattr__(self, key):
+		raise RuntimeError(f'Please install {self._name}')
```

Comment:
Better to keep `AttributeError` and message `.. is required but not installed.`.